app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,7 @@
     android:name=".BoromiApp"
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
-    android:theme="@style/Theme.AppCompat.Light.NoActionBar">
+    android:theme="@style/AppTheme">
```

Comment:
Why is this changed? Changing this will break our current bar

Reply:
It's not actually changing it, I changed AppTheme to Theme.ApppCompat.Light.NoAction bar. This let's android use our defined colours where appropriate.

For example: In the login page, the underline below each text field was the default android color (that dark green color) because we weren't actually applying our theme.

I changed it in this ticket because it was doing the same to the edit_user_fragment

Reply:
o okok